WordPress/Classes/Services/ReaderCardService.swift

```diff
@@ -99,19 +92,20 @@ class ReaderCardService {
 
     /// Remove all cards and saves the context
     func clean() {
-        removeAllCards()
-        coreDataStack.save(syncContext)
+        coreDataStack.performAndSave { context in
+            self.removeAllCards(in: context)
+        }
```

Comment:
Just for fun, here's a different syntax:

```suggestion
        coreDataStack.performAndSave(removeAllCards(in:))
```

IMHO:

- **Pro**: It's more compact
- **Con**: I'm not sure how well it reads

Reply:
If this were Ruby, RuboCop would have suggested making that change. I often run into it, which is why I noticed it here 😅